edx_exams/apps/core/admin.py

```diff
@@ -30,17 +30,20 @@ class ProctoringProviderAdmin(admin.ModelAdmin):
 
 class ExamAdmin(admin.ModelAdmin):
     """ Admin configuration for the Exam model """
-    list_display = ('resource_id', 'course_id', 'provider', 'content_id', 'exam_name',
-                    'exam_type', 'time_limit_mins', 'due_date', 'hide_after_due', 'is_active')
+    list_display = ('course_id', 'provider', 'exam_name', 'exam_type', 'due_date', 'is_active')
```

Comment:
Just curious why `time_limit_mins` and `hide_after_due` aren't included here. Are those fields not included in the current admin pages for edx-proctoring? Or is the information not really relevant?

Reply:
Yeah we don't include these today. This is just the list view with a table of every exam, you can see the attributes when you click into the page for that individual record. I dropped most fields off here since more than a few columns won't fit before scrolling off the the right.